src/net_processing.cpp

```diff
@@ -2309,6 +2311,19 @@ void PeerManagerImpl::ProcessGetCFCheckPt(CNode& peer, CDataStream& vRecv)
     m_connman.PushMessage(&peer, std::move(msg));
 }
 
+void PeerManagerImpl::ProcessBlock(CNode& pfrom, const CChainParams& chainparams, const std::shared_ptr<const CBlock> pblock,
```

Comment:
`chainparams` is not being used in the function body. `m_chainparams` is used however.

Reply:
curious... I wonder why the tests aren't failing then...

Reply:
>  I wonder why the tests aren't failing then

We have no tests to detect unused parameters, IIRC

Reply:
I thought the compiler would at least give a warning - I've seen it warn about unused variables before, so I wonder why it doesn't do this for unused parameters too.